Makefile

```diff
@@ -93,7 +93,8 @@ red=\033[0;31m
 cyan=\033[0;36m
 
 develop: check_venv
-	pip install -e .$(extras) $(packages)
+	- rm src/toil/version.py
```

Comment:
Why is this removing `src/toil/version.py`?

Reply:
This is probably not the best way to do it, but if I put the cwltool version in `version_template.py`, we might need to regenerate `src/toil/version.py` when someone wants to upgrade locally. I could also make it so that `src/toil/version.py` is always regenerated in `setup.py`? 

Suggestions would be appreciated. I don't like how I moved the cwltool version outside of `setup.py`, but I couldn't find another way to access the version inside Toil.